students/TracyA/session04/copy_file.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python
+# Programming in python B Winter 2018
+# February 11, 2017
+# Copy file Lab - Section 4
+# Tracy Allen - git repo https://github.com/tenoverpar/Wi2018-Classroom
+
+
+# File Copy steps
+# Provide a file orginial file name
+source = input("Enter the source file name: ")
+output = input("Enter the destination file name:")
+while (source == output):
```

Comment:
You can also use the sys package to find out if a file already exists.. this might be a more foolproof method.